build.sh

```diff
@@ -544,12 +581,16 @@ function build_cpp()
     ##############################################################################
     # Generate files for srcCpp
 
+    # rtiddsgen ignores any specified rti addional library if using ZeroCopy
+    # Therefore, we need to generate a makefile that contains
+    # nddsmetp and nddssecurity libraries
```

Comment:
I think this comment is the same as the one below.

Reply:
Done